doc/tutorial/analysis.rst

```diff
@@ -0,0 +1,153 @@
+.. ipython:: python
+    :suppress:
+
+    import os
+    import shutil
+
+    import matplotlib as mpl
+    mpl.rcdefaults()
+
+    from veros import tools
+
+    OUTPUT_FILES = tools.get_assets("tutorial_analysis", os.path.join(os.getcwd(), "assets.json"))
+
+
+Analysis of Veros output
+========================
+
+In this tutorial, we will use `IPython <https://ipython.readthedocs.io/en/stable/>`__ or `Jupyter Notebook <https://jupyter.org>`__, `xarray <http://xarray.pydata.org/en/stable/>`__ and `matplotlib <https://matplotlib.org>`__ to load and analyze the model output.
+The analysis below is performed for 100 yr integration of :doc:`global_4deg </reference/setups/4deg>` from the :doc:`setup gallery </reference/setup-gallery>`. The model output is preloaded in our case and it has 4 different files:
+
+.. code:: 
+
+    $HOME/global_4deg/4deg.averages.nc
+    $HOME/global_4deg/4deg.overturning.nc
+    $HOME/global_4deg/4deg.energy.nc
+    $HOME/global_4deg/4deg.snapshot.nc
+
+So, when we load "averages", it means we load ``4deg.averages.nc`` file from our local directory. At the very beginning we will need to load ``xarray``. To do so, execute the following code:
+
+.. ipython:: python
+
+    import xarray as xr
+
+Load and manipulate averages
+----------------------------
+
+.. ipython:: python
+
+    ds = xr.open_dataset(OUTPUT_FILES["averages"], decode_times=False)
+    ds
+
+Let's change the units of geographical coordinates:
+
+.. ipython:: python
+    :suppress:
+
+    ds.xt.attrs["long_name"] = "longitude"
+    ds.xt.attrs["units"] = "deg"
+    ds.yt.attrs["long_name"] = "latitude"
+    ds.yt.attrs["units"] = "deg"
+    ds.zt.attrs["long_name"] = "depth"
+    ds.zt.attrs["units"] = "m"
+
+.. ipython:: python
+
+    ds.xu.attrs["long_name"] = "longitude"
+    ds.xu.attrs["units"] = "deg"
+    ds.yu.attrs["long_name"] = "latitude"
+    ds.yu.attrs["units"] = "deg"
```

Comment:
Opened an issue here: https://github.com/h5netcdf/h5netcdf/issues/116

Reply:
Let's use the `netcdf4` engine for this tutorial instead. All you should need to do is add `netcdf4` to the requirements (it's used by default in xarray). Then you don't need to touch the attributes, and you can decode times.

Reply:
I will do. 
However, I would keep `decode_times=False`, since the decoded time has unit of ns, which is kind of pointless for time span of a year.

Reply:
There are many ways to skin a cat, and your way definitely works, I just feel that since this is an "official" tutorial we should promote best practices when working with xarray. So I changed it to what is recommended in the xarray docs.